src/App.test.js

```diff
@@ -0,0 +1,8 @@
+import { render, screen } from '@testing-library/react';
+import App from './App';
+
+test('renders learn react link', () => {
+  render(<App />);
+  const linkElement = screen.getByText(/learn react/i);
+  expect(linkElement).toBeInTheDocument();
+});
```

Comment:
- [ ] This test is failing please make sure to fix it so that it passes
<img width="516" alt="Screenshot 2023-04-13 at 17 37 56" src="https://user-images.githubusercontent.com/61790587/231840569-c803d946-0ad0-41f6-8741-1805d0a43146.png">